code/modules/power/smes.dm

```diff
@@ -221,18 +227,60 @@
 
 /obj/machinery/power/smes/update_overlays()
 	. = ..()
-	if(machine_stat & BROKEN)
-		return
+	SSvis_overlays.remove_vis_overlay(src, managed_vis_overlays)
```

Comment:
this code becomes redundant if vis_overlays is nuked

Reply:
I have no idea how to make it other way.